src/modules/file/file.usecase.ts

```diff
@@ -64,6 +66,30 @@ export class FileUseCases {
     return files.map((file) => file.toJSON());
   }
 
+  async getByUserAndPlainName(
+    userId: FileAttributes['userId'],
+    plain_name: string,
+    { deleted, page, perPage }: FileOptions = {
+      deleted: false,
+      page: 0,
+      perPage: 5,
+    },
+  ) {
+    const { offset, limit } = Pagination.calculatePagination(page, perPage);
+    const files = await this.fileRepository.findAll({
```

Comment:
Completely discouraged, as you are coupling the **use case**, **which is pure logic**, with the persistence layer (the repository). Ask yourself this:

_What would happen if we change from Sequelize to another ORM?_ Yes, not only the repository layer will be affected by the changes, due to this coupling.